main.go

```diff
@@ -129,6 +129,7 @@ func cmd() *cobra.Command {
 					set(&context.UpgradeCodeMigration)
 				case "pf":
 					set(&context.UpgradePfVersion)
+				case "pulumi":
```

Comment:
It would be really nice to have a comment here that explains what this does. Right now, it looks like `"pulumi"` does nothing.

I'm assuming that this is effetely a way to be able to avoid the `"all"` kind without passing any other kind.